src/Config.cpp

```diff
@@ -79,22 +79,38 @@ Config::Config() {
         bool value = std::any_cast<bool>(elem);
         if (key == "double_quote_to_single_quote") {
             if (value && (node_["single_quote_to_double_quote"]).as<bool>()) {
-                throw logic_error("[ERROR] Configuration value is conflicting with other value");
+                throw logic_error("[ERROR] Configuration value of double_quote_to_single_quote is conflicting with the value of single_quote_to_double_quote");
             }
         } else if (key == "single_quote_to_double_quote") {
             if (value && (node_["double_quote_to_single_quote"]).as<bool>()) {
-                throw logic_error("[ERROR] Configuration value is conflicting with other value");
+                throw logic_error("[ERROR] Configuration value of single_quote_to_double_quote is conflicting with the value of double_quote_to_single_quote");
             }
         }
         return value;
     };
+    auto validate_use_tab = [&](std::string key, std::any elem) {
+        bool value = std::any_cast<bool>(elem);
+        if (value==1 && (node_["tab_width"]).as<int>()==0) {
```

Comment:
Code style is wrong, don't forget to run `clang-format`

Reply:
Just slip of mind after correcting some portion.